plugins/org.eclipse.fordiac.ide.fbtypeeditor.servicesequence/src/org/eclipse/fordiac/ide/fbtypeeditor/servicesequence/handler/NewServiceSequenceHandler.java

```diff
@@ -36,7 +34,7 @@ public Object execute(final ExecutionEvent event) throws ExecutionException {
 		final IStructuredSelection selection = (IStructuredSelection) HandlerUtil.getCurrentSelection(event);
 		final int i = selection.size() - 1;
 		if (i < 0) {
-			return IStatus.ERROR;
+			return null;
```

Comment:
returning null from handlers is considered a bad practice. Better return here Status.CANCEL_STATUS

Reply:
The reason I went to `null` was the documentation of the execute-method:
![image](https://github.com/user-attachments/assets/3c208eb9-4539-4d9a-8e57-72ed9afb5585)

Reply:
Strange! I'm pretty sure that I read somewhere that a handler should return a `Status`. But also the examples in the Eclipse documentation return null. So I should stop replacing in my "clean-ups" to replace the null with Status.